bots/wallet/bot.js

```diff
@@ -5,7 +5,7 @@ function calculateFee(n, tx) {
     }
 
     var gasMultiplicator = Math.pow(1.4, n).toFixed(3);
-    return web3.fromWei(web3.eth.gasPrice * gasMultiplicator * estimatedGas, "ether");
+    return parseFloat(web3.fromWei(web3.eth.gasPrice * gasMultiplicator * estimatedGas, "ether")).toFixed(7);
```

Comment:
I'm a little surprised about fromWei result being a string. According to the [doc](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3fromwei) I would expect a number ? Then parseFloat is supposed to only accept [string](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/parseFloat) argument. 

Or is it a browser quirck? In this case we might want to check the type and parse accordingly.